### PR TITLE
i18n(zh-cn): Create cant-render-page.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create cant-render-page.mdx

#### Related issues & labels (optional)

- #6253

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/cant-render-page.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Astro 无法渲染该路由。
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/cant-render-page.mdx:6]: delete this line

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
标题写错了，zh-cn

File: src/content/docs/zh-cn/reference/errors/cant-render-page.mdx (hunk 1 above)
Comment (huyikai) [reply src/content/docs/zh-cn/reference/errors/cant-render-page.mdx]: Completed.

**Comment by huyikai** [comment]
> 标题写错了，zh-cn

Modifed.

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**